Sources/PackageModel/PackageIdentity.swift

```diff
@@ -0,0 +1,330 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Foundation
+
+import TSCBasic
+import TSCUtility
+
+//@warn_unqualified_access
```

Comment:
Aside: I wish `@warn_unqualified_access` was available for top-level variable declarations (not just functions).